sources/assets/exegol/skel/bloodhound_customqueries/README.txt

```diff
@@ -0,0 +1,11 @@
+If you wish to have your own "Bloodhound customqueries.json" file, you are at the right place.
+
+Two possibilities:
+1/ either you _replace_ Exegol's default provided file,
```

Comment:
Can you use Markdown extension & syntax on your files ? To use the same as the others modules.

Reply:
As mentioned above, this is done.